apps/accounts/automations/base/manager.py

```diff
@@ -154,15 +148,7 @@ def wait_and_save_recorder(self, recorder, max_retries=10, retry_interval=2):
         return False
 
     def save_record(self, recorder):
-        if self.is_running_in_celery():
-            self.wait_and_save_recorder(recorder)
-        else:
-            thread = threading.Thread(
-                target=self.wait_and_save_recorder,
-                args=(recorder,),
-                daemon=True
-            )
-            thread.start()
+        self.wait_and_save_recorder(recorder)
 
     def on_host_success(self, host, result):
 
```

Comment:
I cannot provide feedback about the code without seeing the actual text of the code. Please share the code so I can assist you further.